src/Components/AdvancedSearch/Internal/QueryBuilder/QueryBuilderUtils.ts

```diff
@@ -0,0 +1,174 @@
+    if (type === 'boolean') {
+        return 'True';
+    } else {
+        return '0';
+    }
+};
+
+export const DEFAULT_OPERATOR: OperatorData = {
+    operatorType: 'Simple',
+    operatorSymbol: '='
+};
+
+export const DEFAULT_COMBINATOR = 'And';
+
+export const getOperators = (
+    propertyType?: PropertyValueType
+): IDropdownOption<OperatorData>[] => {
+    if (!propertyType) {
+        return [];
+    }
+
+    const operators: IDropdownOption[] = [
+        {
+            key: '1',
+            text: 'Equals',
+            data: {
+                operatorType: 'Simple',
+                operatorSymbol: '='
+            }
+        },
+        {
+            key: '2',
+            text: 'Not equals',
+            data: {
+                operatorType: 'Simple',
+                operatorSymbol: '!='
+            }
+        }
+    ];
+    switch (propertyType) {
+        case 'string':
+            operators.push({
+                key: '3',
+                text: 'Contains',
+                data: {
+                    operatorType: 'Function',
+                    operatorFunction: (property: string, value: string) => {
+                        return `CONTAINS(T.${property}, ${value})`;
+                    }
+                }
+            });
+            operators.push({
+                key: '4',
+                text: 'Not contains',
+                data: {
+                    operatorType: 'Function',
+                    operatorFunction: (property: string, value: string) => {
+                        return `NOT CONTAINS(T.${property}, ${value})`;
+                    }
+                }
+            });
+            break;
+        case 'float':
+        case 'integer':
+        case 'double':
+        case 'long':
+            operators.push({
+                key: '3',
+                text: 'Greater than',
+                data: {
+                    operatorType: 'Simple',
+                    operatorSymbol: '>'
+                }
+            });
+            operators.push({
+                key: '4',
+                text: 'Less than',
+                data: {
+                    operatorType: 'Simple',
+                    operatorSymbol: '<'
+                }
+            });
+            operators.push({
+                key: '5',
+                text: 'Greater or equal',
+                data: {
+                    operatorType: 'Simple',
+                    operatorSymbol: '>='
+                }
+            });
+            operators.push({
+                key: '6',
+                text: 'Less or equal',
+                data: {
+                    operatorType: 'Simple',
+                    operatorSymbol: '<='
+                }
+            });
+            break;
+        default:
+    }
+    return operators;
+};
+
+export const buildQuery = (querySnippets: QueryRowData[]) => {
+    let fullQuery = `SELECT *\nFROM DIGITALTWINS T\n`;
```

Comment:
should we add the model id into the query as well? Because in the property dropdown it is sectioned by models and if there are same named properties from two different models I would expect from the result to show me only the twins of that model which has that property, is that right?

Reply:
I'll look into this on the next PR, that actually queries for data